payments_service/braintree/tests/test_views.py

```diff
@@ -66,6 +79,32 @@ def test_override_active_flag(self):
         call_args = self.solitude.braintree.mozilla.paymethod.get.call_args
         eq_(call_args[1]['active'], '0')
 
+    def test_patch_inactive(self):
+        res, pay_methods = self.get()
+        pay_method = pay_methods[0]
+
+        resource = mock.Mock()
+        resource.get.return_value = {}
+        resource.patch.return_value = {}
+        self.solitude.braintree.mozilla.paymethod.return_value = resource
+        res = self.client.patch(pay_method['resource_uri'],
+                                data=json.dumps({'active': 0}),
```

Comment:
That should really be false, but 0 works. Its a boolean field, so Django applies bool(...) to the value and stores it in the db. So anything falsy in Python works, e.g.: 0 or false.

I'll change to be more consistent though.